src/metabase/search/scoring.clj

```diff
@@ -244,21 +258,11 @@
 (defn weights-and-scores
   "Default weights and scores for a given result."
   [result]
-  [{:weight 2
-    :score  (pinned-score result)
-    :name   "pinned"}
-   {:weight 2
-    :score  (bookmarked-score result)
-    :name   "bookmarked"}
-   {:weight 3/2
-    :score  (recency-score result)
-    :name   "recency"}
-   {:weight 1
-    :score  (dashboard-count-score result)
-    :name   "dashboard"}
-   {:weight 1/2
-    :score  (model-score result)
-    :name   "model"}])
+  [{:weight 2 :score (pinned-score result) :name "pinned"}
+   {:weight 2 :score (bookmarked-score result) :name "bookmarked"}
+   {:weight 3/2 :score (recency-score result) :name "recency"}
```

Comment:
I moved the other weights to integers, I think we agree that it's aesthetically pleasing. The reason I could do that is because I force the "text-*" scorers total weight to be exactly 10. Meaning the weights I changed are only relative to each other. But doubling these weights would reduce the signal coming from the "text-*" scorers, so I left them unchanged.